### ISSUE TITLE
Select with default value causes error on rollback

### ISSUE THREAD

**Issue by lukegalea** [emberjs/data#2972]
See http://emberjs.jsbin.com/bifojaqiqu/1/edit or https://gist.github.com/anonymous/b009ffe56ae8ccb81ddc

If you choose a non default value, then rollback - you will get something like this:

     Attempted to handle event `didSetProperty` on <AdminApp.AttachmentElement:ember1144:11654> while in state root.deleted.saved. Called with {name: position, oldValue: 3, originalValue: 3, value: 5}.

**Comment by tomdale** [comment]
This should actually be fixed in Glimmer, since we do not propagate changes to the view layer immediately but rather mark them as dirty and wait for the next update. Let's wait and see if this issue goes away once Glimmer is merged in.

**Comment by igorT** [comment]
@lukegalea did you have a chance to test glimmer out?

**Comment by sly7-7** [comment]
@igorT I just updated the jsbin with ember canary, and ember-data beta, the same thing happen when choosing a value, then rollback, then choosing an other value.

http://emberjs.jsbin.com/zewuloneki/1/edit

**Comment by igorT** [comment]
@sly7-7 thanks

**Comment by jcope2013** [comment]
still seeing issue (ember 1.13.10/ember data 1.13.13)

**Comment by fivetanley** [comment]
As Ember.Select is deprecated, I'm inclined to close. If this can be reproduced some other way, feel free to reopen or open a new issue. We won't be doing backports to 1.13 much longer I imagine.
